app/ts/library/utils.ts

```diff
@@ -7,3 +7,19 @@ export function addressString(address: bigint): string {
 }
 
 export const min = (left: bigint, right: bigint) => left < right ? left : right
+
+function bigintToUint8Array(value: bigint, numberOfBytes: number) {
+	if (typeof value === 'number') value = BigInt(value)
+	if (value >= 2n ** BigInt(numberOfBytes * 8) || value < 0n) throw new Error(`Cannot fit ${ value } into a ${ numberOfBytes }-byte unsigned integer.`)
+	const result = new Uint8Array(numberOfBytes)
+	for (let i = 0; i < result.length; ++i) {
+		result[i] = Number((value >> BigInt(numberOfBytes - i - 1) * 8n) & 0xffn)
+	}
+	return result
+}
```

Comment:
I'm assuming this was just copied from somewhere without touching internals?

Reply:
yeah, from interceptor